assets/src/block-validation/error/index.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { PanelBody, Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	AMP_VALIDITY_REST_FIELD_NAME,
+	VALIDATION_ERROR_ACK_ACCEPTED_STATUS,
+	VALIDATION_ERROR_ACK_REJECTED_STATUS,
+} from '../constants';
+import { NewTabIcon } from '../icon';
+import { ErrorPanelTitle } from './error-panel-title';
+import { ErrorContent } from './error-content';
+
+/**
+ * Component rendering an individual error. Parent component is a <ul>.
+ *
+ * @param {Object} props Component props.
+ * @param {string} props.clientId
+ * @param {number} props.status
+ * @param {number} props.term_id
+ */
+export function Error( { clientId, status, term_id: termId, ...props } ) {
+	const { selectBlock } = useDispatch( 'core/block-editor' );
+	const { review_link: reviewLink } = useSelect( ( select ) => select( 'core/editor' )?.getEditedPostAttribute( AMP_VALIDITY_REST_FIELD_NAME ) ) || {};
```

Comment:
![image](https://user-images.githubusercontent.com/134745/102928646-f3e56600-444d-11eb-9641-4d26102ff0aa.png)

```suggestion
	const { review_link: reviewLink } = useSelect( ( select ) => select( 'core/editor' )?.getEditedPostAttribute( AMP_VALIDITY_REST_FIELD_NAME ), [] ) || {};
```